optimize/backend/pom.xml

```diff
@@ -385,19 +427,23 @@
         </exclusion>
       </exclusions>
     </dependency>
+    -->
 
     <!-- Opensearch  -->
+    <!-- unused/declared
     <dependency>
       <groupId>org.apache.httpcomponents.core5</groupId>
       <artifactId>httpcore5</artifactId>
       <version>${apache.http5-core.version}</version>
     </dependency>
-
+    -->
+    <!-- unused/declared
     <dependency>
       <groupId>org.apache.httpcomponents.core5</groupId>
       <artifactId>httpcore5-h2</artifactId>
       <version>${apache.http5-core.version}</version>
     </dependency>
```

Comment:
possibly also http2 related

Reply:
This dependency was not added with my http2 work for Optimize. @grlimacan - I saw you added this dependency for OPT-7226, OPT-7195 and OPT-7197 for OpenSearch work

Reply:
Yep, we need that for the OpenSearch client. Is thee a problem?